README.md

```diff
@@ -1,2 +1,158 @@
-# helm-mirror
+# Helm Mirror Plugin
+
 Helm plugin used to mirror repositories
+
+## Usage
+
+```shell
+Mirror Helm Charts from an index file into a local folder.
+
+For example:
+
+helm-mirror https://yourorg.com/charts /yourorg/charts
+
+This will download the index file and the charts into
```

Comment:
If we make use of Chart Museum there is no need to fetch the raw index file. Simply start the binary and drop tgzs into your specified charts directory, and they can be served from your localhost. Read about it here. https://github.com/helm/chartmuseum

Reply:
You could also mirror to a web server using flat files or object storage. Would there be an option for multiple systems?